en/dashboard/README.md

```diff
@@ -1,16 +1,20 @@
 # Dashboard
 
-Mautic 1.4.0 brought a customizable dashboard where each user can compose widgets with information she/he wants to track.  Mautic 2.0 brought a number of enhancements to the Dashboard.
+Mautic 1.4.0 brought a customizable dashboard where each user can compose widgets with information they want to track.  Mautic 2.0 brought a number of enhancements to the Dashboard.
 
 ## Date range filter
 
-All the widgets will display data in the selected global date range filter at the top of the widget list. The default date range is set from 30 days ago to today. Line charts will change the time unit automatically depending on the day count selected in the date range filter like this:
+All the widgets will display data in the selected global date range filter at the top of the widget list. The default date range is set from 30 days ago to today. 
```

Comment:
I'm not sure this is true. There may be exceptions. For example the *Upcoming emails* widget does not take the selected date range filter into account AFAIK.

Reply:
I think it says that later on at line 15?

Reply:
You're right! I missed that.